sobel/makefile

```
@@ -0,0 +1,11 @@
+sobel: main.o sobel.o checker
+	gcc -fsanitize=address -lm main.o sobel.o -o sobel
```

I recommend adding couple more flags, FYI: https://developers.redhat.com/blog/2018/03/21/compiler-and-linker-flags-gcc
```suggestion
	gcc -Wall -Wextra -Wcast-allign -fstack-protector -fsanitize=address -lm main.o sobel.o -o sobel
```